ingestion/functions/parsing/common/python/parsing_lib/input_event.json

```diff
@@ -4,11 +4,17 @@
     "s3Key": "5f0b9a7ead3a2b003edc0e7f/2020/07/12/2320/content.json",
     "sourceUrl": "https://foo.bar",
     "sourceId": "5f0b9a7ead3a2b003edc0e7f",
-    "uploadIds": ["123456789012345678901234"],
+    "uploadIds": [
+        "123456789012345678901234"
+    ],
     "dateFilter": {
         "numDaysBeforeToday": 3,
         "op": "LT"
     },
+    "dateRange": {
```

Comment:
should this be allowed with dateFilter set as well? seems contradictory

Reply:
SG, I've added a check.